packages/medusa/src/services/pricing.ts

```diff
@@ -135,10 +135,9 @@ class PricingService extends TransactionBaseService {
         })
       )
 
-      taxedPricing.calculated_price_incl_tax =
-        variantPricing.calculated_price_includes_tax
-          ? variantPricing.calculated_price
-          : variantPricing.calculated_price + taxedPricing.calculated_tax
+      taxedPricing.calculated_price_incl_tax = variantPricing.calculated_price_includes_tax
```

Comment:
comment(non blocking): my linter does not arrange it that way. is that coming from my conf?
```
      taxedPricing.calculated_price_incl_tax =
        variantPricing.calculated_price_includes_tax
          ? variantPricing.calculated_price
          : variantPricing.calculated_price + taxedPricing.calculated_tax
```

Reply:
it the same below

Reply:
My prettier is doing that.
Double check ESLint and Prettier are in my checklist. At some point we have to normalize all the files in the project.